nuitka/distutils/bdist_nuitka.py

```diff
@@ -158,9 +170,7 @@ def _buildPackage(self, build_lib):
         else:
             # Delete the entire source copy of the module
             shutil.rmtree(
-                os.path.join(
-                    self.build_lib, self.main_package.replace(".", os.path.sep)
-                )
+                os.path.join(self.build_lib, self.main_target.replace(".", os.path.sep))
```

Comment:
Ah ok, for packages this is a thing, but not for modules. For those just delete it as a file, and I guess, the `.py` must be protected from replacing maybe. Trace what do you here, and do the right thing.

But what also strikes me as wrong, is that not all the packages, and not all the module files are deleted, but just the main one. It should be done for all or none I guess.  We could really use Nuitka to learn to `--replace` when it compiles, i.e. delete the input files included. Doing it on the outside is kind of harder to do, but doable, so lets stick to it for now.

For now, this code will probably have issues when multiple packages are included, which is something people do, then I think things might survive. On the other hand, are not Python files all deleted anyway? Can you check if this delete here is even needed?

Reply:
There are no .py files in the build folder even without this delete for package example_1, that means this is unnecessary, correct?